lib/bitcoin/lock/MongoDbLockTransactionStore.ts

```diff
@@ -16,7 +18,8 @@ export default class MongoDbLockTransactionStore {
    * @param serverUrl The target server url.
    * @param databaseName The database name where the collection should be saved.
    */
-  public constructor (private serverUrl: string, private databaseName: string) {
+  public constructor (private serverUrl: string, private databaseName?: string) {

```

Comment:
Since the defaultDatabaseName is not used anywhere else, couldn't the constructor just be

`public constructor (private serverUrl: string, private databaseName: string = 'sidetree')`?